src/runboat/k8s.py

```diff
@@ -175,17 +180,17 @@ def make_deployment_vars(
         build_env=settings.build_env | build_settings.env,
         build_secret_env=settings.build_secret_env | build_settings.secret_env,
         build_template_vars=settings.build_template_vars | build_settings.template_vars,
+        kubefiles_path=kubefiles_path,
     )
 
 
 @contextmanager
 def _render_kubefiles(deployment_vars: DeploymentVars) -> Generator[Path, None, None]:
-    with resources.path(
-        __package__, "kubefiles"
-    ) as kubefiles_path, tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
So if you accept `kubefiles_path: str | None` as an argument to `_render_kubefiles`, you can make your own context manager that yields it if set, or yields `resource.path` otherwise.